airgun/widgets.py

```diff
@@ -2090,3 +2091,48 @@ class Link(Text):
     def fill(self, value):
         if value:
             self.browser.click(self)
+
+
+class PopOverWidget(GenericLocatorWidget):
+    """Popover-content UI widget which contains header, drop_down or input_box and
+    submit button. This is associated within a table.
+
+    Example html representation::
+
+        <h3 class="popover-title">Authorize login delegation (Default: false)</h3>
+            <form class="form-inline editableform" style="">
+                <select class="form-control input-sm">
+                    <option value="0">No</option>
+                    <option value="1">Yes</option></select>
+                <div class="editable-buttons">
+                    <button type="submit" class="btn btn-primary btn-sm editable-submit">
+
+        <h3 class="popover-title">Authorize login delegation auth ...(Default: Not set)</h3>
+            <form class="form-inline editableform" style="">
+                <input type="text" ...>
+                <div class="editable-buttons">
+                    <button type="submit" class="btn btn-primary btn-sm editable-submit">
+
+    Locator example::
+
+        //div[contains(@class, 'editable-open')]
+    """
+    column_text = Text("//span[contains(@class, 'editable-click')]")
+    header = Text("//*[contains(@class, 'popover-title')]")
```

Comment:
Try to avoid `//*` if you can specify exact locator

Reply:
I changed because of your comment mentioned here https://github.com/SatelliteQE/airgun/pull/334#discussion_r290263646. Do you mean I wanted to use tag instead of '*' and use contains more likely?

Reply:
I mean relative paths `.//`, not the `//*` =)